<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Participante</class>
 <widget class="QDialog" name="Participante">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>681</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Participante</string>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>108</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Participante</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>42</y>
     <width>16</width>
     <height>589</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>40</y>
     <width>239</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Informations sur la participante</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nom : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>130</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>173</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>190</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>220</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Province : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>250</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Téléphone 1 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Téléphone 2 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_annee_naissance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Année de naissance : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_personne_nourries">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>119</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Personnes nourries : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_consentement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>132</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Consentement photo : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>73</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Appelation : </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbx_photo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>282</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>La participante consent à être prise en photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>280</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Courriel : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_appelation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mme.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Organisme</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>École</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Municipalité</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Employée</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Autre</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txt_adresse1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>Adresse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_adresse2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Adresse 2 (facultatif)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ville">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ville</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_province">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>177</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Québec</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alberta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Colombie-Britannique</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Île-du-Prince-Édouard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Manitoba</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nouveau-Brunswick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nouvelle-Écosse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nunavut</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ontario</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Saskatchewan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Terre-Neuve-et-Labrador</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Territoires du Nord-Ouest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yukon</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txt_code_postal">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Code postal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_telephone1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>Téléphone 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_poste1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Poste</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_telephone2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>Téléphone 2 (facultatif)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_poste2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Poste</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbx_annee_naissance">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>60</width>
     <height>26</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbx_personnes_nourries">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>46</width>
     <height>26</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Courriel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_prenom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>3</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nom">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>5</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_transaction">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>68</y>
     <width>350</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prix</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lbl_transaction">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Transactions</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_inscription">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Inscription</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_inscription">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>350</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Heure</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lbl_membre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Status de membre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>640</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>640</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_membre">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>499</y>
     <width>74</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Membre</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_numero_membre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de membre :  </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_renouvellement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de renouvellement : </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_actif">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>500</y>
     <width>52</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Actif</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_honoraire">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="statusTip">
    <string>Activer le status de membre honoraire pour ce membre.</string>
   </property>
   <property name="text">
    <string>Honoraire</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="ded_renouvellement">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>560</y>
     <width>98</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="displayFormat">
    <string notr="true">dd-MM-yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_renew">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>560</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Renouveler</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_numero_membre">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>108</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Numéro de membre</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbx_appelation</tabstop>
  <tabstop>txt_prenom</tabstop>
  <tabstop>txt_nom</tabstop>
  <tabstop>txt_adresse1</tabstop>
  <tabstop>txt_adresse2</tabstop>
  <tabstop>txt_ville</tabstop>
  <tabstop>cbx_province</tabstop>
  <tabstop>txt_code_postal</tabstop>
  <tabstop>txt_email</tabstop>
  <tabstop>txt_telephone1</tabstop>
  <tabstop>txt_poste1</tabstop>
  <tabstop>txt_telephone2</tabstop>
  <tabstop>txt_poste2</tabstop>
  <tabstop>sbx_annee_naissance</tabstop>
  <tabstop>sbx_personnes_nourries</tabstop>
  <tabstop>cbx_photo</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_cancel</tabstop>
  <tabstop>chk_honoraire</tabstop>
  <tabstop>ded_renouvellement</tabstop>
  <tabstop>chk_actif</tabstop>
  <tabstop>chk_membre</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
